test/javascripts/components/chat-message-collapser-test.js

```diff
@@ -8,12 +8,31 @@ import {
   query,
 } from "discourse/tests/helpers/qunit-helpers";
 
+const youtubeCooked =
+  "<p>written text</p>" +
+  '<div class="onebox lazyYT-container" data-youtube-id="ytId1" data-youtube-title="Cats are great">Vid 1</div>' +
+  "<p>more written text</p>" +
+  '<div class="onebox lazyYT-container" data-youtube-id="ytId2" data-youtube-title="Kittens are great">Vid 2</div>' +
+  "<p>and even more</p>";
+
+const animatedImageCooked =
+  "<p>written text</p>" +
+  '<p><img src="lesource1" class="animated onebox"></img></p>' +
+  "<p>more written text</p>" +
+  '<p><img src="lesource2" class="animated onebox"></img></p>' +
+  "<p>and even more</p>";
+
+const externalImageCooked =
+  "<p>written text</p>" +
+  '<p><a href="http://cat1.com" class="onebox"></a></p>' +
```

Comment:
Related to comment above, would be good to have an `img` inside the anchor here.

Reply:
Caught it!!